src/gapp_login/wechat/endpoints.py

```diff
@@ -80,6 +80,10 @@ async def login_wechat(
         rv = await auth.create_authorization_response(request, user, ctx)
         if rv.status_code >= 400:
             tx.raise_rollback()
+        rv_location = rv.headers["location"]
+        if rv_location.startswith("wxa://"):
+            params = parse.parse_qs(rv_location.split("?", 1)[1])
+            return auth.handle_response(200, dict(code=params.get("code")), {})
```

Comment:
should be fine to return just a `dict` innstance

Reply:
right, fixed